src/utils/db/types.rs

```diff
@@ -476,6 +479,18 @@ impl FromRow<'_, PgRow> for ProofCompressionJobInfo {
     }
 }
 
+pub(crate) fn proof_generation_time_from_row(
```

Comment:
Instead of having this function you can implement `FromRow` for `ProofGenerationTime`.